src/handle.jl

```diff
@@ -33,6 +33,7 @@ immutable Handle{T <: AbstractSundialsObject}
     ptr_ref::Ref{Ptr{T}} # pointer to a pointer
 
     @compat function (::Type{Handle}){T}(ptr::Ptr{T})
+        T <: Void && throw(ArgumentError("Handle assigned to a null pointer"))
```

Comment:
>Wouldn't ptr == C_NULL be a better check (C_NULL == Ptr{Void}(0) == Ptr{Int}(0) gives true)?

I think it's the same thing? But I changed it since `ptr == C_NULL` matches the rest of the Sundials code better.